src/style.css

```diff
@@ -1 +1,83 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  height: 60px;
+  /* border: solid 1px #000; */
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+}
+
+.nav-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-link {
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.nav-link:active {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
```

Comment:
create variable with color that you use more than 1 time, and use variable instead in all places